src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="Logo image"
+        >
+      </a>
```

Comment:
Add spacing between a and nav, generally take a look again at your intend you have some errors like on tag `a` on line 34. Again go through the checklist

Reply:
I formatted correct anchors.